h/models/group_scope.py

```diff
@@ -33,5 +40,14 @@ class GroupScope(Base):
     #: http://localhost:5000
     origin = sa.Column(sa.UnicodeText, nullable=False)
 
+    #: A path which, concatenated with ``origin``, creates a wildcarded prefix
+    #: against which URLs may be compared for scope. This allows for scope
+    #: granularity at a path level (instead of just origin).
+    #: e.g. for a group with scope origin ``https://foo.com`` and a path ``/bar``:
+    #:
+    #: * ``https://foo.com/bar/baz.html`` in scope
+    #: * ``https://foo.com/ding/foo.html`` NOT in scope
+    path = sa.Column(sa.UnicodeText, nullable=True)
```

Comment:
I think `nullable` is `True` by default?

Reply:
Is it OK to be explicit, though?